data/hai/hai culture conversations.txt

```diff
@@ -207,3 +207,57 @@ mission "Gambling Judgment"
 			label end
 			`	He takes your order diligently and with a smile.`
 				decline
+
+
+
+mission "Academy Motto"
+	minor
+	source
+		government "Hai"
+		attributes "urban"
+		not attributes "station"
+	to offer
+		random < 1
+		has "language: Wanderer"
+	on offer
+		conversation
+			`	On this particular day, the spaceport is crowded beyond belief by some arcane confluence of chance and logistics. Despite having resolved the bulk of your business the port authorities have informed you that there is no safe departure slot open for at least seven hours (and even that's not a promise.) Leaving you alone to fill in your unexpected bit of time in the relatively early morning.`
+			choice
+				`	(Play games on your ship.)`
+				`	(Have a look around further afield.)`
+					goto fieldtrip
+			`	Not interested in getting lost in an expansive Hai city, you simply load up an entertainment console to keep you occupied for the next few hours.`
+				decline
+			label fieldtrip
+			`	The tendency of the Hai to efficiently pack people into vibrant but densely populated spaces is on full display as you navigate through throngs of Hai who mostly make polite efforts to avoid bumping into you. Colorful banners, flags, murals and even windows all provide brilliant highlights to the oft-utilitarian architectural styles underlying them. Holoprojected welcome signs even grace the entry-ways of several apartment buildings that you pass, with caveats for door-to-door salespeople.`
+			`	One apartment complex's exterior fencing bears a very large sign which your translator informs you says, "No Meme Content Here Please".`
+			`	After some time you find what appears to be a cafe, or equivalent, and sit down where you can get a good view of one of the more impressive building complexes around. After a moment a holoprojection built into the table asks you in human standard whether you'd like to order through the table or converse with a member of staff.`
```

Comment:
Is 'human standard' ever established to be a thing? Not really a problem, but you might want to make it more clear what it means.

Reply:
I'm like 90% sure it's referred to somewhere else.

Reply:
Alright then, feel free to resolve